### PR TITLE
dlgOpenFromLibrary help on datasets

### PR DESCRIPTION
**Pull request by maxwellfundi**

### CODE REVIEW

**Comment by maxwellfundi** [comment]
fixes #1934

File: instat/dlgFromLibrary.vb [hunk 1]
```diff
@@ -37,6 +37,7 @@ Public Class dlgFromLibrary
 
     Private Sub SetDefaults()
         rdoDefaultDatasets.Checked = True
+        cmdHelp.Enabled = False
```
Comment (dannyparsons) [inline instat/dlgFromLibrary.vb]: You shouldn't have this in SetDefaults, the enabled should always be controlled by the checks in the sub you created. If you need to do this here, call the check sub instead.

Comment (maxwellfundi) [inline instat/dlgFromLibrary.vb]: Oo okay. I need it to be disabled by default. So that makes sense

File: instat/dlgFromLibrary.vb [hunk 2]
```diff
@@ -155,4 +157,20 @@ Public Class dlgFromLibrary
             frmMain.clsRLink.RunScript(clsDataFunction.ToScript(), strComment:=ucrBase.strComment)
         End If
     End Sub
+
+    Private Sub cmdHelp_Click(sender As Object, e As EventArgs) Handles cmdHelp.Click
+        Dim clsHelp As New RFunction
+        clsHelp.SetRCommand("help")
+        clsHelp.AddParameter("topic", Chr(34) & lstCollection.SelectedItems(0).Text & Chr(34))
+        clsHelp.AddParameter("help_type", Chr(34) & "html" & Chr(34))
+        frmMain.clsRLink.RunScript(clsHelp.ToScript)
```
Comment (dannyparsons) [inline instat/dlgFromLibrary.vb]: Can you also add to this so that there is a comment with this command

Comment (maxwellfundi) [inline instat/dlgFromLibrary.vb]: Do you mean i add a comment here to explain whats the command is? or what needs to be added?

Comment (dannyparsons) [inline instat/dlgFromLibrary.vb]: No I mean add to the RunScript function so that a comment appears in the Output window above the command when it's run

**Comment by maxwellfundi** [comment]
@dannyparsons  thats fixed now

File: instat/dlgFromLibrary.vb [hunk 3]
```diff
@@ -163,7 +163,7 @@ Public Class dlgFromLibrary
         clsHelp.SetRCommand("help")
         clsHelp.AddParameter("topic", Chr(34) & lstCollection.SelectedItems(0).Text & Chr(34))
         clsHelp.AddParameter("help_type", Chr(34) & "html" & Chr(34))
-        frmMain.clsRLink.RunScript(clsHelp.ToScript)
+        frmMain.clsRLink.RunScript(clsHelp.ToScript, strComment:="Running help for the selected dataset")
```
Comment (dannyparsons) [inline instat/dlgFromLibrary.vb]: Good, but this should include the dialog name where the command has come from as well. 
Also "running help", I don't really understand what that means. "getting help for..." or "opening help page for..." makes more sense.

**Pull request merged**